cityofaustin/atd-moped
# Prevent saving of map edits if there are no map features

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10991

This PR updates the edit component map flow to enable removing all features from a component and to disable the **Save Edit** button when that happens. The button enables again when another feature is added.

## Testing
**URL to test:** 

local or https://md-10991-prevent-save--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Create a new project and then add a new line or point component
2. Add some features and then save it
3. Edit the component that you just created and delete every feature by deselecting or using the trash can on drawn features
4. You should see the **Save Edit** button disable when you have no features left
5. Select a feature from the map or drawn one
6. The **Save Edit** button should be enabled again and you can save the update

---
#### Ship list
